src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -47,30 +71,46 @@ private FieldBuilder(EventStreamFieldType fieldType)
             return new FieldBuilder<TSourceType, TReturnType>(fieldType);
         }
 
+        /// <summary>
+        /// Sets the graph type of the field.
+        /// </summary>
+        /// <param name="type">The graph type of the field.</param>
         public virtual FieldBuilder<TSourceType, TReturnType> Type(IGraphType type)
         {
             FieldType.ResolvedType = type;
             return this;
         }
 
+        /// <summary>
+        /// Sets the name of the field.
+        /// </summary>
         public virtual FieldBuilder<TSourceType, TReturnType> Name(string name)
         {
             FieldType.Name = name;
             return this;
         }
 
+        /// <summary>
+        /// Sets the description of the field.
+        /// </summary>
         public virtual FieldBuilder<TSourceType, TReturnType> Description(string description)
         {
             FieldType.Description = description;
             return this;
         }
 
+        /// <summary>
+        /// Sets the deprecation reason of the field.
+        /// </summary>
         public virtual FieldBuilder<TSourceType, TReturnType> DeprecationReason(string deprecationReason)
         {
             FieldType.DeprecationReason = deprecationReason;
             return this;
         }
 
+        /// <summary>
+        /// Sets the default value of fields on input object graph types.
```

Comment:
I suggest to throw `InvalidOperationException` if `FieldType` is not an input type.

Reply:
I agree in principle.  I'm not sure exactly how this would work in reality, since it's the graph type that's an input type, not the field, and since the builder doesn't know what graph type the field is connected to, or even if it has been added to a graph type at all.  If you wish to explore further, I suggest you open a bug report for this issue and we can solve it in a separate PR.

Reply:
Let's keep it as is for now.